lib/cc/pull_requests.rb

```diff
@@ -1,8 +1,34 @@
 class CC::PullRequests < CC::Service
+  VALID_TOKEN_MESSAGE = "Access token is valid.".freeze
+  CANT_POST_COMMENTS_MESSAGE = "Access token is invalid - can't post comments".freeze
+  CANT_UPDATE_STATUS_MESSAGE = "Access token is invalid - can't update status.".freeze
+  INVALID_TOKEN_MESSAGE = "Access token is invalid.".freeze
+
   def receive_test
     setup_http
 
-    receive_test_status
+    able_to_update_status = able_to_update_status?
+
+    able_to_post_comments =
```

Comment:
I continue to not entirely trust extension test coverage behavior. I saw a test that hits this method, didn't I? But the extension shows most lines in here as uncovered.